recipes-multimedia/aamp/aampmetrics_git.bb

```diff
@@ -0,0 +1,23 @@
+SUMMARY = "AAMP Metris library"
+DESCRIPTION = "Metrics Library is an independent library for session statistics generation/reporting in JSON format"
+HOMEPAGE = "https://github.com/rdkcmf/rdk-aampmetrics"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=175792518e4ac015ab6696d16c4f607e"
+
+DEPENDS_append = " cjson"
+
+PV = "0.1.gitr${SRCPV}"
+
+SRC_URI = "git://github.com/rdkcmf/rdk-aampmetrics.git;branch=stable2;protocol=https"
+SRCREV = "c6b9de63f2a8755db59ee300c02c6c89e392dad5"
+
+S = "${WORKDIR}/git"
+
+inherit cmake
+
+PACKAGES = "${PN} ${PN}-dev ${PN}-dbg"
```

Comment:
Is this needed? I think these packages should be enabled by default. No need to specify them explicitly

Reply:
It seems some more additional things are getting added and if I remove this it force be to add INSANE_SKIP_${PN}-dev = "dev-elf" and finally wpeframework-plugins sendup with wpeframework-plugins rdepends on aamp-dev [dev-deps]

Reply:
ok, then please leave it like it is

Reply:
done